### PR TITLE
Add tweets, logo, and oldest experiments

### PR DESCRIPTION
**Pull request by NirantK**

### CODE REVIEW

File: CleanlabDistilbert.ipynb [hunk 1]
```diff
@@ -0,0 +1,634 @@
+{
```
Comment (meghanabhange) [inline CleanlabDistilbert.ipynb:1]: <p>The notebook doesn't follow PEP8 style guide, is that okay?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (meghanabhange) [inline CleanlabDistilbert.ipynb:1]: <p>The paths are specific to colab only (Maybe loading using gdown would help?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: ULMFIT.ipynb [hunk 2]
```diff
@@ -0,0 +1,976 @@
+{
```
Comment (meghanabhange) [inline ULMFIT.ipynb:1]: <p>Colab specific paths</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: data_tfms.ipynb [hunk 3]
```diff
@@ -39,7 +39,7 @@
   },
```
Comment (meghanabhange) [inline data_tfms.ipynb:39]: <p><code>trail.text</code> is missing from data folder</p><pre class="ql-syntax" spellcheck="false">trial_df = parse_file("data/raw/trial.txt"); len(trial_df)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (meghanabhange) [inline data_tfms.ipynb:39]: <pre class="ql-syntax" spellcheck="false">trial_df.to_json("data/raw/trial.json", orient="records")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (meghanabhange) [inline data_tfms.ipynb:39]: <p><code>train.text</code> missing from data folder</p><pre class="ql-syntax" spellcheck="false">train_df = parse_file("data/raw/train.txt"); len(train_df)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (meghanabhange) [inline data_tfms.ipynb:39]: <pre class="ql-syntax" spellcheck="false">train_df.to_json("data/raw/train.json", orient="records")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (meghanabhange) [inline data_tfms.ipynb:39]: <pre class="ql-syntax" spellcheck="false">trial = pd.read_json("data/raw/trial.json")
train = pd.read_json("data/raw/train.json")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (meghanabhange) [inline data_tfms.ipynb:39]: <pre class="ql-syntax" spellcheck="false">df.to_json("data/interim/train-large.json", orient="records")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

File: download.ipynb [hunk 4]
```diff
@@ -40,6 +40,33 @@
     "get_files_from_gdrive(\"https://drive.google.com/file/d/1T_AvUYwc8Fld_8VoUj9s9S1ZbmpkAsVI/view?usp=sharing\", \"trial.txt\")\n",
```
Comment (meghanabhange) [inline download.ipynb:40]: <pre class="ql-syntax" spellcheck="false">get_files_from_gdrive("https://drive.google.com/file/d/1T_AvUYwc8Fld_8VoUj9s9S1ZbmpkAsVI/view?usp=sharing", "data/raw/trial.txt")
get_files_from_gdrive("https://drive.google.com/file/d/1fzhNn_baYWE6Ltxf1VKlkYzCUwUlHVXx/view?usp=sharing", "data/raw/train.txt")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: CleanlabDistilbert.ipynb (hunk 1 above)
Comment (NirantK) [reply CleanlabDistilbert.ipynb:1]: <p>Yeah, we'll fix that later. Add this as a TODO in the README on the repo. </p><p> </p>

Comment (NirantK) [reply CleanlabDistilbert.ipynb:1]: <p>Yes. We <code>gdown</code>  this please?</p>

File: ULMFIT.ipynb (hunk 2 above)
Comment (NirantK) [reply ULMFIT.ipynb:1]: <p>Convert all Drive paths to public paths via <code>gdown</code>  please?</p>

File: data_tfms.ipynb (hunk 3 above)
Comment (NirantK) [reply data_tfms.ipynb:39]: <p>Which file was this? Do we have a copy of this somewhere?</p>

Comment (NirantK) [reply data_tfms.ipynb:39]: <p>PR it? </p>

Comment (NirantK) [reply data_tfms.ipynb:39]: <p>PR it when you are updating paths? PLIJ</p><p> </p>

Comment (NirantK) [reply data_tfms.ipynb:39]: <p>PR it when you are updating paths? PLIJ PLIJ PLIZ</p>

File: CleanlabDistilbert.ipynb (hunk 1 above)
Comment (meghanabhange) [reply CleanlabDistilbert.ipynb:1]: Resolved by 8763287

File: data_tfms.ipynb (hunk 3 above)
Comment (meghanabhange) [reply data_tfms.ipynb:39]: Downloaded by `download.ipynb`

File: download.ipynb (hunk 4 above)
Comment (meghanabhange) [reply download.ipynb]: resolved by 4ec581d

File: data_tfms.ipynb (hunk 3 above)
Comment (meghanabhange) [reply data_tfms.ipynb:39]: file added by 4ec581d

Comment (meghanabhange) [reply data_tfms.ipynb:39]: file added by 4ec581d

Comment (meghanabhange) [reply data_tfms.ipynb:39]: file added by 4ec581d

Comment (meghanabhange) [reply data_tfms.ipynb:39]: Resolved by 6885478

Comment (meghanabhange) [reply data_tfms.ipynb:39]: Resolved by 6885478

Comment (meghanabhange) [reply data_tfms.ipynb:39]: Resolved by 6885478

Comment (meghanabhange) [reply data_tfms.ipynb:39]: Resolved by 6885478

Comment (meghanabhange) [reply data_tfms.ipynb:39]: Resolved by 6885478

Comment (meghanabhange) [reply data_tfms.ipynb:39]: Resolved by 6885478

File: CleanlabDistilbert.ipynb (hunk 1 above)
Comment (meghanabhange) [reply CleanlabDistilbert.ipynb:1]: Did most of the notebooks now :)

0eb3691
c5253a8
6f6c2dd
f5a7a13
9a9a0ad

Comment (NirantK) [reply CleanlabDistilbert.ipynb:1]: You the best 👏

**Pull request merged**